modules/core/arrow-core-data/src/test/kotlin/arrow/core/IorTest.kt

```diff
@@ -140,3 +149,16 @@ class IorTest : UnitSpec() {
     }
   }
 }
+
+private fun <A> Gen.Companion.ior(gen: Gen<A>) =
+  object : Gen<Kind<ForIor, A>> {
+    override fun constants(): Iterable<Kind<ForIor, A>> =
+      gen.constants().map {
+        Ior.Both(it, it) as Kind<ForIor, A>
+      }
+
+    override fun random(): Sequence<Kind<ForIor, A>> =
+      gen.random().map {
+        Ior.Both(it, it) as Kind<ForIor, A>
+      }
+  }
```

Comment:
This is not a good generator. A good generator should cover all cases of `Ior` and thus require two generators as input. But tbh wait with this untill the zip pr is merged and just use `LiftGen`

Reply:
true, i went ahead and changed it. the code later can be moved to GenK